src/components/PillBadge/PillBadge.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { string } from 'prop-types';
+import glamorous from 'glamorous';
+import constants from '../../constants';
+
+const { spacing, fontSize, lineHeight, color } = constants;
+
+const PillBadgeContainer = glamorous.span(props => ({
+   display: 'inline-block',
+   padding: `${spacing[0]} ${spacing[1]}`,
+   fontSize: fontSize[0],
+   fontWeight: 700,
+   lineHeight: lineHeight.none,
+   textTransform: 'uppercase',
+   color: props.color,
+   backgroundColor: props.background,
+   borderRadius: 9999,
+}));
+
+const PillBadge = props => <PillBadgeContainer {...props} />;
```

Comment:
What's the advantage of just naming the result of `glamorous.span` `PillBadge` instead of having the extra functional component? I don't see a problem but I'm just curious if that's a pattern.

Reply:
I'd definitely prefer to just export that glamorous component directly but we have to do it like this so `react-docgen` will recognize it as a react component.

Reply:
Oh cool, that's good to know. I just wanted to make sure glamorous.span didn't return some type of Component-ish thing.